From 97af210a5f3f14bf2ab632e954a2f9ab22e9abd5 Mon Sep 17 00:00:00 2001
From: velsinki <112010-velsinki@users.noreply.gitlab.gnome.org>
Date: Tue, 17 Oct 2023 23:13:23 +0200
Subject: [PATCH] Revert "window: Parse toast title as plain text"

This reverts commit 06adcd8a54a60452796c93e5de8585feb97649f2.

It required libadwaita API which is not available in GNOME 44
---
 src/nautilus-window.c | 1 +
 1 file changed, 1 deletion(-)

diff --git a/src/nautilus-window.c b/src/nautilus-window.c
index 3fdec0d7ca..0033dcf5d5 100644
--- a/src/nautilus-window.c
+++ b/src/nautilus-window.c
@@ -1313,7 +1313,6 @@ nautilus_window_show_operation_notification (NautilusWindow *window,
 
     toast = adw_toast_new (main_label);
     adw_toast_set_priority (toast, ADW_TOAST_PRIORITY_HIGH);
-    adw_toast_set_use_markup (toast, FALSE);
 
     current_location = nautilus_window_slot_get_location (window->active_slot);
     if (gtk_window_is_active (GTK_WINDOW (window)))
-- 
GitLab

